README.md

```diff
@@ -60,11 +61,12 @@ See [Architecture](https://github.com/open-switch/opx-docs/wiki/Architecture) fo
 - `cps\_send\_events`
 
 ### cps\_model\_info 
+
 This tool is useful to get all information about CPS objects on the target. It is used to get the attributes of a specific CPS object, or first-level contents of a given YANG path of the CPS object (as defined in the YANG model).
 
 #### Usage
 
-`cps\_model\_info` *CPS Object Path as defined in the YANG model*
+`cps\_model\_info` — CPS Object Path as defined in the YANG model
```

Comment:
@amybuck can you also fix these instances of backslashes (`\`) inside these inline code blocks? Outside of a code block, underscores have to be escaped using `\_`, but inside of a code block, underscores do _not_ need to be escaped. It might make more sense if you view the actual rendered README.md file, and you'll see a bunch of places where the cps scripts have backslashes where they don't need to, like `cps\_model\_info`, whereas they should have underscores but _NOT_ backslashes. (`cps\_model\_info` should just be `cps_model_info`). There are a bunch of places in the file where this occurs.